app/selenium_ui/conftest.py

```diff
@@ -58,6 +58,12 @@ def print_timing(interaction=None):
     def deco_wrapper(func):
         @functools.wraps(func)
         def wrapper():
+            global fail_login
+            if LOGIN_ACTION_NAME in interaction:
+                fail_login = False
```

Comment:
fail_login sound like a method to fail login. Let's change the variable name to 'login_failed'.